### ISSUE TITLE
FROST Library Tracking Issue

### ISSUE THREAD

**Issue by kayabaNerve** [serai-dex/serai#111]
Tracking issue for the FROST library. Tasks are required if they must be done before the library hits 1.0, where 1.0 means the library is audited and usable by Serai at mainnet. Issues are included based on impact, even if they aren't directly relevant to the library.

Issue | Required | Status                | Notes                                         |
------|----------|-----------------------|-----------------------------------------------|
#9    | Yes      | Partially Done        |                                               |
#80   | No       | Not Started           | Could increase security and fault tolerance   |
#95   | No       | Not Started           | Security increase                             |
#124  | No       | Not Started           |                                               |
#151  | No       | Not Started           |                                               |
#195  | No       | Not Started           | Literally a 2.0 discussion                    |
#264  | Yes      | Not Started           |                                               |

If any issue is closed, it'll be removed from the table. If a new issue is created, it should be added.

Additional tasks:

- Examples.

**Comment by kayabaNerve** [comment]
Removed several issues related to the (separated out) DKG crate (#56 #132 #149). That probably should have its own tracking issue...

**Comment by kayabaNerve** [comment]
Also moved out #42.

**Comment by kayabaNerve** [comment]
This was audited in #199.

**Comment by kayabaNerve** [comment]
Given FROST's stability, and this being post-audit, I'm fine closing the tracking issue for individual issues as they occur.
